main.go

```diff
@@ -164,14 +156,8 @@ func generateThumbnail(w http.ResponseWriter, rmethod, rpath string, sourceURL s
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		copyHeader(w.Header(), resp.Header)
-		io.Copy(w, resp.Body)
-		return
```

Comment:
doesn't make sense to proxy back an invalid result, dunno why I would have done that